server/views/pages/about.njk

```diff
@@ -36,7 +36,7 @@
             href: "/create-goal/accommodation",
             classes: "govuk-button--secondary",
             type: 'button'
-        }
+        } if data.readWrite === true
```

Comment:
is the `=== true` needed in this case?

Reply:
No, but I quite like super-explicit checks for permission-based validation - I can take it out if you prefer.